.ctignore

```diff
@@ -0,0 +1 @@
+charts/datadog/ci/
```

Comment:
Let's keep the `/ci` as it is and remove this file

Reply:
yes, forgot to push its removal
<img width="450" alt="image" src="https://github.com/user-attachments/assets/87e87487-d041-4791-b6e9-3f412762f01c" />